bundle/manifests/limitador-operator.clusterserviceversion.yaml

```diff
@@ -37,7 +37,7 @@ metadata:
     capabilities: Basic Install
     operators.operatorframework.io/builder: operator-sdk-v1.14.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-  name: limitador-operator.v0.0.1
+  name: limitador-operator.v0.2.0
```

Comment:
IMO `main` branch should be `0.0.0`. Each release branch needs to update version and `replace` or `skip` or `skipRange` fields

Reply:
Updated to always be 0.0.0 on main